aspnet/fundamentals/localization.rst

```diff
@@ -0,0 +1,337 @@
+ASP.NET Localization
+====================
+
+By `Tom Archer`_
+
+The ASP.NET 5 localization support provides middleware for establishing the correct culture and UI culture on the thread based on the request, and mechanisms for accessing localized content based on the current culture. In this article, you'll learn how to enable localization and establish the culture using both built-in culture providers. You'll also see how to implement your own custom culture provider.
```

Comment:
Can we start off with some statements that are a little less implementation specific? Something like: "The ASP.NET 5 localization support enables you to customize your application for a given culture and locale. Localization consists primarily of translating the user interface. The ASP.NET 5 localization support . . . "